google_shopping.js

```diff
@@ -1,5 +1,54 @@
 var data = require("./products.json")
 
 // Write your solutions below
+var q = 0
+for (i = 0; i < data["items"].length; i++){
+
+	if (data["items"][i].product.inventories[0].availability === "inStock"){
+		q++;
+	}
+	
+}
+console.log("There are " + q + " items In Stock.");
+
+var q = 0;
+for (i = 0; i < data["items"].length; i++){
+	if (data["items"][i].product.images.length > 1){
+		q++;		
+	}
+}
+console.log("There are " + q + " items with more than one image link.");
+
+var q = 0;
+for (i = 0; i < data["items"].length; i++){
+	if (data["items"][i].product.brand.includes("anon")){
+		q++;		
+	}
+}
+console.log("There are " + q + " Canon branded items.");
+
+var q = 0;
+for (i = 0; i < data["items"].length; i++){
+	if (data["items"][i].product.author.name.includes("eBay") && data["items"][i].product.brand.includes("anon")){
+		q++;		
```

Comment:
My only other suggestion would be to use more descriptive variables for your counters, so that your code is easily read and interpreted by others. Ie:

```var productsInStock = 0```
```var multipleImages = 0```
```var canonProducts = 0```